core/acl/aclservice.go

```diff
@@ -195,6 +197,37 @@ func (a *aclService) ChangePermissions(ctx context.Context, spaceId string, perm
 	return nil
 }
 
+func (a *aclService) ApproveLeave(ctx context.Context, spaceId string, identities []crypto.PubKey) error {
+	sp, err := a.spaceService.Get(ctx, spaceId)
+	if err != nil {
+		return err
+	}
+	acl := sp.CommonSpace().Acl()
+	acl.RLock()
+	st := acl.AclState()
+	identitiesMap := map[string]struct{}{}
+	for _, identity := range identities {
+		identitiesMap[identity.Account()] = struct{}{}
+	}
+	for _, rec := range st.RemoveRecords() {
+		for _, identity := range identities {
+			if rec.RequestIdentity.Equals(identity) {
+				delete(identitiesMap, identity.Account())
+			}
+		}
+	}
+	if len(identitiesMap) != 0 {
+		acl.RUnlock()
+		var identitiesString string
+		for identity := range identitiesMap {
+			identitiesString += identity + ", "
```

Comment:
Stringbuffer?

Reply:
This is just an error to print the identities :-) It doesn't matter

Reply:
But it is better to just do strings with separator though

Reply:
Changed that